File: patch_syntax.py
with open('./backend/ui/static/js/rolloverTool.js', 'r') as f:
    content = f.read()

content = content.replace("    }\n\n    exportMatrixCSV: function() {", "    },\n\n    exportMatrixCSV: function() {")

with open('./backend/ui/static/js/rolloverTool.js', 'w') as f:
    f.write(content)

File: patch_syntax.js
const fs = require('fs');
let js = fs.readFileSync("backend/ui/static/js/rolloverTool.js", "utf8");

// Fix syntax error - add missing comma before exportMatrixCSV if needed.
js = js.replace("    }\n\n    exportMatrixCSV", "    },\n\n    exportMatrixCSV");

fs.writeFileSync("backend/ui/static/js/rolloverTool.js", js);
